src/typography.css

```diff
@@ -596,6 +603,7 @@ textarea {
  * <div class='txt-spacing2'>spaced out</div>
  * <div class='txt-spacing4'>way spaced out</div>
  */
+.txt-spacing1 { letter-spacing: 0.1em !important; }
```

Comment:
Make sure to note this somehow, maybe in the commit message, so we don't miss it in the changelog next release. I've been prepping the changelog by looking at commit messages.